app/views/homepage/_promo_banner_example.html.erb

```diff
@@ -0,0 +1,6 @@
+<div id="homepage-promo-banner">
+  <div class="banner-message">
+    <p><strong>Some Title</strong> On some date something has happened.
+    <a href="https://something.gov.uk" rel="external nofollow">More&nbsp;information</a></p>
```

Comment:
Done. Thanks for pointing that out, I'll make sure we have copy for the hidden part